.github/workflows/llvm-to-smt.yml

```diff
@@ -54,11 +53,10 @@ jobs:
           python3 generate_encodings.py \
```

Comment:
It might be useful to print the command line invocation of generate_encodings.py to the workflow log, 
 so that one can quickly see what command line was used to generate the encodings, and even run that locally. Right now, as I understand this isn't printed anywhere.

Reply:
It is showed here:
![image](https://github.com/user-attachments/assets/3c2acd60-e6e3-4fd1-b7b0-1ba92f87e741)

Do you mean we should show the commit (and kernel version)? I think that's a good point. We can probably modify `generate_encodings.py` to print its own call command.

Note I'm not planning to merge this pull request. I'm just using it to generate encodings that I'll then use in https://github.com/bpfverif/agni/pull/33. I might also send a couple commits from here as separate pull request (ex. a less verbose mode).

Reply:
> It is showed here:

Aaah, it was hidden under the collapsible triangle. I see it now. 

> Do you mean we should show the commit (and kernel version)? I think that's a good point. We can probably modify generate_encodings.py to print its own call command.

Yeah. It would be useful to be able to just copy-paste and try out locally (with some minor edits). While the above output is good enough, the `--commit` and `--kernver` are still missing. 

> We can probably modify generate_encodings.py to print its own call command.

Can we perhaps have the workflow echo this to the log after it has resolved all the variable names?